modules/init.gradle

```diff
@@ -6,6 +6,14 @@ import org.gradle.internal.hash.HashValue
 if (System.properties["http.proxyHost"] == "squid.lax.liferay.com") {
 	logger.lifecycle "Using proxy {}:{}.", System.properties["http.proxyHost"], System.properties["http.proxyPort"]
 
+	File repositoryLocalDir = file("../.m2-tmp")
```

Comment:
If we add this code to `init.gradle`, then we must use it in `build-buildscript.gradle` and everywhere else we deal with `.m2-tmp`. My personal suggestion is to leave `init.gradle` alone and to move this logic to `project-templates/build.gradle` 😋